src/lib/core/sio.c

```diff
@@ -55,38 +55,49 @@ static_assert(SMALL_STATIC_SIZE > NI_MAXHOST + NI_MAXSERV,
  * checks and all.
```

Comment:
Please fix the typo in the commit title:
displaing -> displaying

Reply:
Thanks :) Fixed.